docs/user/requirements.rst

```diff
@@ -67,7 +61,7 @@ To update Scipy and install netCDF4 using Conda, the following code is used:
 Optional Packages
 -----------------
 
-Optionally, you can install timezonefinder to allow for automatic timezone detection when performing Enviornment Analysis.
+Optionally, you can install timezonefinder to allow for automatic timezone detection when performing Environment Analysis.
```

Comment:
I think this should include the other optional packages too

Reply:
Solved here: https://github.com/RocketPy-Team/RocketPy/pull/368/commits/8a45c7b896e5ef869ef2072b22e18e1e86f5e18a